### DISCUSSION TITLE
unable to parse .env file

### DISCUSSION THREAD

**Discussion by Kavita-lab** [socioboard/Socioboard-5.0#314] [General]
Hii, while generating key 'composer install & php artisan key:generate' ,I am getting this error ,any suggestion to fix it  given that environment file name  is .env .
 
> @php artisan package:discover --ansi
The environment file is invalid!
Failed to parse dotenv file. Encountered unexpected whitespace at [<<php domain>>].
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

**Comment by sureshbabu1995** [comment]
Hi @Kavita-lab 

Can you please share the .env file, so that I can check the issue, some where you missing the .env standards
